src/main/java/core/basesyntax/Palindrome.java

```diff
@@ -21,6 +21,23 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
-        return false;
+        String reverse;
+        String wholeStr;
+        String[] tokens = text.toLowerCase()
```

Comment:
@olehedza this solution is to complex. Can we simplify? 
1. You use regexp to replace the symbols

Reply:
Done!